Source/HtmlTextWriter.Tests/HtmlTextWriterTests.cs

```diff
@@ -394,5 +394,186 @@ public void CheckNotDisposed()
             const string test = "<html />\r\n";
             Assert.AreEqual(test, html);
         }
+        
+        [TestMethod]
+        public void TestScenario3()
+        {
+            string bodyBackgroundColor = "#E6FFFF";
+            string bodyBorderColor = "#C1CDC1";
+            string fontFamily = "Arial";
+            string bodyFontSize = "13";
+            string TestHeaderBackgroundColor = "#FFF59D";
+            string bodyResultFontSize = "15";
+            string TestResultStatusPass = "#66FF66";
+            string TestResultStatusFail = "#FF0000";
+
+            MemoryStream mem = new MemoryStream();
+            StreamWriter sw = new StreamWriter(mem);
+            HtmlTextWriter writer = new HtmlTextWriter(sw);
+
+            //Open HTML
```

Comment:
As a general nit here, can you fix the comments to be `//<space>Uppercase setence` consistently throughout?